pegasus/sites.v3/code.org/public/pd-workshop-survey/splat.haml

```diff
@@ -314,14 +314,17 @@ title: "Professional Development Workshop Survey"
 
   #thanks{style: 'display: none;'}
     %h1 Thank you for your submission
+    %h3 We appreciate your feedback.
 
 - if mimeo_url
   #materials-order{style: 'display: none;'}
+    %h1 Order Supplies
     %p
-      Click
-      %a{href: mimeo_url} here
-      if you'd like to be connected with our partner Mimeo to order free materials for the unplugged
+      Click below if you'd like to be connected with our partner Mimeo to order free materials for the unplugged
       lessons in CS Fundamentals, or to check on an existing order (US attendees only).
+    %p
+      %a{href: mimeo_url}
```

Comment:
I couldn't get it to work, given that there's a button instead of text inside the <a> element. Also note this is in pegasus, so it doesn't have all of the Rails functionality. It does still have link_to, but whenever I try anything besides `= link_to 'display text', 'destination'` it doesn't show up.